service/src/main/java/bio/terra/landingzone/library/landingzones/management/ResourcesDeleteManager.java

```diff
@@ -148,18 +145,20 @@ private GenericResource deleteResource(ResourceToDelete resourceToDelete) {
       logger.info("Resource deleted. id:{}", resourceToDelete.privateEndpoint().id());
     }
 
-    if (resourceToDelete.solution() != null) {
-      logger.info(
-          "Deleting landing zone solution {} for resource: {}",
-          resourceToDelete.solution().id(),
-          resourceToDelete.resource().id());
+    if (resourceToDelete.solutions() != null && !resourceToDelete.solutions().isEmpty()) {
+      resourceToDelete
```

Comment:
Can you explain a bit more about the change here? I am not grokking the switch to the `solutions()` iteration.